components/programmes/ConditionsOfJoining.tsx

```diff
@@ -11,6 +12,8 @@ export function ConditionsOfJoiningField({
   conditionsOfJoining,

```

Comment:
It is recommended to have your filename match your component name in order to improve code readability (e.g. helps other devs know at a glance they are dealing with the parent component in the file as the file often contains a few exported components/functions) and to keep thing consistent across the app. 2 choices: rename file to ConditionsOfJoiningField or component function to ConditionsOfJoining.